antlr/src/main/antlr4/org/apache/iotdb/db/qp/sql/SqlBase.g4

```diff
@@ -84,6 +84,7 @@ statement
     | KILL QUERY INT? #killQuery
     | TRACING ON #tracingOn
     | TRACING OFF #tracingOff
+    | TRACING CLEAR ALL #tracingClearAll
```

Comment:
hi , thanks for your contribution.
you need add some UT or IT for your function.
Thanks

Reply:
Added UT and IT tests